docs/administration/back_office/subitems_list.md

```diff
@@ -1,47 +1,86 @@
 ---
-description: Inject a sub-items list into your back office customizations.
+description: Inject a sub-items list into your back office customizations or customize the view.
 ---
 
 # Sub-items list
 
 The Sub-items List module is meant to be used as a part of the editorial interface of [[= product_name =]].
 It provides an interface for listing the sub-items of any location.
 
-!!! caution
+## Add custom sub-items list view
+
+You can extend the Sub-items List module to add your own views to it.
```

Comment:
`or override the existing view`?
If you pass the existing viewName as a first parameter in registerView(), you will override the existing view with your own. https://github.com/ibexa/admin-ui/blob/main/src/bundle/ui-dev/src/modules/sub-items/services/view.registry.js#L26
I think it is worth mentioning because this was the original customer problem.

Reply:
Thanks, added in https://github.com/ibexa/documentation-developer/pull/2735/commits/34c544213a8143fd390cad5a29b2da6fe321973f